qutebrowser/browser/hints.py

```diff
@@ -179,6 +180,7 @@ class HintContext:
     group = attr.ib(None)
     hint_mode = attr.ib(None)
     first = attr.ib(False)
+    extended_filter_targets = False
```

Comment:
Why save this instead of just pulling it from config directly?

Reply:
Removed.